app/controllers/books_controller.rb

```diff
@@ -0,0 +1,11 @@
+class BooksController < ApplicationController
+  include Wor::Paginate
+  before_action :authenticate_user!
+  def show
+    @book = Book.find(params[:id])
```

Comment:
What happens when params don't have a param called `:id` ? I prefer to use `params.require(:id)` and if in request don't have id param raise error :smile:

Reply:
I applied the changes